code.md

```diff
@@ -30,9 +30,9 @@ adds support to upstream AOSP for several boards including:
 * Snapdragon(tm) 8 Gen 2 Mobile Hardware Development Kit (HDK 8550)
 * Hikey 960
 
-BayLibre have support packages for
+BayLibre have support packages for:
 
-* MediaTek Genio 350-EVK: [https://baylibre.pages.baylibre.com/mediatek/rita/device/mediatek/boards/mtk-android-14/index.html](https://baylibre.pages.baylibre.com/mediatek/rita/device/mediatek/boards/mtk-android-14/index.html)
+* MediaTek Genio 700, 510 and 350 EVKs: [https://baylibre.pages.baylibre.com/mediatek/rita/device/mediatek/boards/mtk-android-14/index.html](https://baylibre.pages.baylibre.com/mediatek/rita/device/mediatek/boards/mtk-android-14/index.html)
```

Comment:
Thanks for the contribution. +1

I just checked the link and the link is broken. It shows a 404. Do you mind to also fix the link in this PR?

Reply:
hmm something is odd. When I open the new link I see a login page:

![image](https://github.com/user-attachments/assets/56e8ea38-1b26-406d-9a73-e3acfe1a9b3b)